src/core/control/settings/Settings.cpp

```diff
@@ -224,6 +229,42 @@ void Settings::loadDefault() {
     /**/
 }
 
+auto Settings::loadViewMode(size_t mode) -> bool {
```

Comment:
Use `enum ViewMode` instaed of `size_t`

Reply:
Same as first [change-request](https://github.com/xournalpp/xournalpp/pull/4514#discussion_r1057764129)